core/database/src/main/java/com/mifos/room/entities/accounts/savings/DepositType.kt

```diff
@@ -20,13 +20,13 @@ import kotlinx.parcelize.Parcelize
 @Parcelize
 data class DepositType(
     @PrimaryKey
-    var id: Int? = null,
+    val id: Int? = null,
 
     @ColumnInfo(name = "code")
-    var code: String? = null,
+    val code: String? = null,
 
     @ColumnInfo(name = "value")
-    var value: String? = null,
+    val value: String? = null,
```

Comment:
@itsPronay 
Just a brief reminder you can actually remove the @ColumnInfo(name = "value") annotation here since the column name will default to the property name value anyway. So, without it, the table column will still be named value, and your code will look a bit cleaner!